src/main/scala/com/comcast/xfinity/sirius/uberstore/segmented/SegmentedUberStore.scala

```diff
@@ -286,16 +291,16 @@ class SegmentedUberStore private[segmented] (base: JFile, eventsPerSegment: Long
    */
   private def replaceSegments(replacementMap: Map[Segment, String]) = replaceLock.synchronized {
     for (segment <- replacementMap.keys) {
-      replaceSegment(segment, replacementMap(segment))
+      replacesegment(segment, replacementMap(segment))
```

Comment:
I think the `find/replace` got carried away on these, don't need to lower-case the `S` in segment here and in the two below